test/classes-additional.coffee

```diff
@@ -0,0 +1,301 @@
+# Classes
+# -------
+
+# * Class Instantiation
+# * Inheritance and Super
+# * Default parameters
+# * Version Compatability
+test "multiple super calls", ->
+  # class A 
```

Comment:
Multiple `super`s will generate a runtime in JS, so this should be able to use `throws`.